CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+# 4.4.0 / 2022-10-25
+
+### New Features
+
+- Added Spanish on the welcome page
+- Added Norwegian to the supported languages
+- Updated translations : Spanish, French, English, Portuguese
+- UX / UI improvements (sidebar, header, news)
+- UX / UI improvements on small screens
+- Updated sidebar colors
+- Video player for video previews
+- Modified comments are now easily identifiable
+- Featuring a new borderless design
+- Added code blocks to TinyMCE
+- Improved pertinence of notifications about tasks
+
+For technical users:
+- Added curl example on api documentation
+- Improved documentation
+
```

Comment:
I think we can simplify a little, here a suggestion

```suggestion
- Featuring a new borderless design
- Modified comments are now easily identifiable
- Code preview on advanced search
- The user interface is now available in Norwegian 
- Video player for video previews
- Improved pertinence of notifications about tasks
- UX / UI improvements (sidebar, header, news, mobile)
```
I also changed the order from most important to less important (imo).